andela/orophin-cfh
# #151980423 Czar Should Draw Card on Game Start

## What does this PR do?
It adds modal on the game screen for Czar to choose black. Before now, the game automatically chooses card for czar, this pull request fixes the issue.

## Description of Task to be completed?
- create modal modal for czar to choose card

## How should this be manually tested?
- login/signup into the app or play as guest
- invite friends to play (when players are upto 3 or more)
- click on "start game", the first modal confirms game start
- Czar is greeted with a modal for him/her to choose black card (see screenshots)

#### Any background context you want to provide?
## What are the relevant pivotal tracker stories?
[CZARs should be able to draw cards by clicking](https://www.pivotaltracker.com/story/show/151980423)
#### Screenshots (if appropriate)
<img width="1405" alt="screen shot 2017-11-03 at 4 11 44 pm" src="https://user-images.githubusercontent.com/31279497/32380961-c39ef6aa-c0b1-11e7-922a-ccd9dbb5c59c.png">

#### Questions
